insanely_fast_whisper_rocm/webui/ui.py

```
@@ -202,35 +320,140 @@ def _process_transcription_request_wrapper(
+
+    def _run_transcription() -> None:
+        """Run transcription in background and signal stream completion."""
+        nonlocal final_result, final_error
+        try:
+            final_result = process_transcription_request(
+                audio_paths=audio_paths,
+                transcription_config=transcription_cfg,
+                file_handling_config=file_handling_cfg,
+                progress_tracker=cast(Any, progress_proxy),
+            )
+        except Exception as exc:  # pragma: no cover - defensive passthrough
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)